include/constants.h

```diff
@@ -1,8 +1,10 @@
 #pragma once
 
+#include "storage_EXPORTS.h"
+
 namespace azure {  namespace storage_lite {  namespace constants {
 
-#define DAT(x, y) extern const char *x; const int x ## _size{ sizeof(y) / sizeof(char) - 1 };
+#define DAT(x, y) extern AZURE_STORAGE_API const char *x; const int x ## _size{ sizeof(y) / sizeof(char) - 1 };
```

Comment:
>AZURE_STORAGE_API [](start = 25, length = 17)

Is this used in adls? Is that the reason of this being exported as well?

Reply:
Yes, apparently ADLS needs to use those constant strings.